hack/patch-precache-image.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/python3
+import yaml
+import sys
+
+file = sys.argv[1]
+pull = sys.argv[2]
+with open(file,'r') as f:
+    obj = yaml.safe_load(f)
+for item in obj.get('spec',{}).get('relatedImages',{}):
```

Comment:
is it assured there will always be a relatedImages field?

Reply:
Yes, because of this:
https://github.com/openshift-kni/cluster-group-upgrades-operator/blob/cb2f0bcd5f0b61bd686603485f2d38c7a793ae55/config/manifests/pre-cache-image-patch.yaml#L7